test/unit-test/FreeRTOS_DNS_Parser/FreeRTOS_DNS_Parser_utest.c

```diff
@@ -80,6 +80,9 @@ static void dns_callback( const char * pcName,
 
 extern BaseType_t xBufferAllocFixedSize;
 
+extern pucAddrBuffer[ 2 ];
```

Comment:
Please follow the file format in _utest.c

Reply:
Updated.